### PR TITLE
API docs: use Dart sample for NgZone

### LINKED ISSUE #267: NgZone documentation has TypeScript sample code
https://webdev.dartlang.org/angular/api/angular2.core/NgZone-class

### PR DESCRIPTION
**Pull request by chalin**
Fixes #267

Full example committed via https://github.com/dart-lang/site-webdev/pull/508

cc @kwalrath

### CODE REVIEW

File: lib/src/core/zone/ng_zone.dart [hunk 1]
```diff
@@ -11,70 +15,66 @@ import 'package:stack_trace/stack_trace.dart';
...
+///     print('Current progress: $progress%');
+///     if (progress < 100) {
+///       new Future<Null>.delayed(const Duration(milliseconds: 10),
+///           () => _increaseProgress(doneCallback));
 ///     } else {
 ///       doneCallback();
 ///     }
 ///   }
 /// }
+///
```
Comment (kwalrath) [inline lib/src/core/zone/ng_zone.dart]: we should remove this blank line, no?

**Comment by kwalrath** [comment]
@matanlurey any idea why https://travis-ci.org/dart-lang/angular2/jobs/217031298 failed?

File: lib/src/core/zone/ng_zone.dart (hunk 1 above)
Comment (chalin) [inline lib/src/core/zone/ng_zone.dart]: I had noticed that too. The fragment creation script is meant to clean trailing blank lines so I'm unsure what happened in this case. I have a note to look at the scripts again to see why this happened.

Comment (chalin) [inline lib/src/core/zone/ng_zone.dart]: Done

**Comment by kevmoo** [comment]
@kwalrath – seems these are just new errors with the latest Dart

See https://travis-ci.org/dart-lang/angular2/jobs/216538815 on the master branch...

**Comment by kevmoo** [comment]
@kwalrath + @matanlurey filed an issue against analyzer: https://github.com/dart-lang/sdk/issues/29221

**Comment by chalin** [comment]
Anything needed before this is merged?

**Comment by kevmoo** [comment]
Nope!

**Pull request closed without merging**